modules/loader/src/main/scala/com/snowplowanalytics/snowplow/rdbloader/dsl/VacuumScheduling.scala

```diff
@@ -0,0 +1,53 @@
+package com.snowplowanalytics.snowplow.rdbloader.dsl
+
+import cats.ApplicativeError
+import fs2.Stream
+import retry._
+import retry.syntax.all._
+import cats.syntax.all._
+import cats.effect.{Concurrent, MonadThrow, Timer}
+import com.snowplowanalytics.snowplow.rdbloader.config.{Config, StorageTarget}
+import com.snowplowanalytics.snowplow.rdbloader.db.Statement
+import eu.timepit.fs2cron.ScheduledStreams
+import eu.timepit.fs2cron.cron4s.Cron4sScheduler
+
+import scala.concurrent.duration._
+
+object VacuumScheduling {
+
+  def retryPolicy[F[_]: Timer: Concurrent]: RetryPolicy[F] =
+    RetryPolicies.fibonacciBackoff[F](1.minute) join RetryPolicies.limitRetries[F](10)
```

Comment:
After 10 attempts, this will fail. But I believe the 10th failure will crash the app -- that feels wrong to me, as we are trying to make RDB Loader as resilient as possible against crashing.  A failure to vacuum is not critical to loading.

Is it possible please to instead handle the final error, log it, and then ignore it?

For bonus points... can you think of how we can surface information about vacuum events, either success or failure?  Maybe a new statsd metric?  Or maybe a webhook?

We probably don't need to raise an alarm if the vacuum fails - that would be too noisy for the pipeline operator.  But when debugging slow loading, it will be very important to know if recent vacuums were successful.

Reply:
I'm fairly certain you still haven't solved the problem of the app crashing after 10 failed attempts.  I can see you've changed the `onError` callback.  But this doesn't stop cats retry from calling `ME.raiseError(error)` on the final attempt: [see here in cats-retry repo](https://github.com/cb372/cats-retry/blob/v2.1.1/modules/core/shared/src/main/scala/retry/package.scala#L77)